src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=${PROD_DB_URL}
+spring.datasource.username=${PROD_DB_USERNAME}
+spring.datasource.password=${PROD_DB_PASSWORD}
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
Checked with Suvi, and it seems it would be practicable to use update instead of create-drop for the production environment. But that's the only comment I have :)